toolkit/tools/precacher/precacher.go

```diff
@@ -207,14 +235,18 @@ func createChroot(workerTar, chrootDir string, leaveChrootOnDisk bool) (queryChr
 		return
 	}
 
-	// Install the repoquery package from upstream
+	// Install the repoquery package from upstream, then clean up any existing repos
 	logger.Log.Infof("Installing '%s' package to get 'repoquery' command", dnfUtilsPackageName)
-	queryChroot.Run(func() error {
-		_, err = installutils.TdnfInstall(dnfUtilsPackageName, rootDir)
-		if err != nil {
-			err = fmt.Errorf("failed to install '%s':\n%w", dnfUtilsPackageName, err)
+	err = queryChroot.Run(func() error {
+		_, chrootErr := installutils.TdnfInstall(dnfUtilsPackageName, rootDir)
+		if chrootErr != nil {
+			chrootErr = fmt.Errorf("failed to install '%s':\n%w", dnfUtilsPackageName, err)
+			return chrootErr
 		}
-		return err
+
+		// Remove all existing repos, we will be adding the repo files we want to query later
+		chrootErr = os.RemoveAll("/etc/yum.repos.d")
```

Comment:
Does the installer chroot get reset each time a tool runs? (Doing this won't affect other tools, correct?)

Reply:
Each time we create a chroot we get a fresh copy based on the `.tar.gz` file that contains the chroot.